android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayout.java

```diff
@@ -419,6 +421,23 @@ public void OnNotificationAdded(String id, int type, long timestamp,
       mBraveRewardsNativeWorker.GetAllNotifications();
   }
 
+  private boolean mayShowBraveAdsOobeDialog() {
```

Comment:
oobe?

Reply:
Address in https://github.com/brave/brave-core/pull/3905/commits/7e8eb45cf1dff475909b9c4e89d609f4d33194a5 as suggested.